src/components/Table/Formatters/selectionHeaderCellFormatter.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table } from '../index';
+
+const selectionHeaderCellFormatter = ({
+  cellProps,
+  column,
+  rows,
+  onSelectAllRows
+}) => {
+  const unselectedRows = rows.filter(r => !r.selected).length > 0;
+  return (
+    <Table.SelectionHeading aria-label={column.header.label} {...cellProps}>
+      <Table.Checkbox
+        id="selectAll"
+        label="Select all rows"
```

Comment:
Should use a customizable label here.

Reply:
good catch! will update this later today...

Reply:
fixed